conbench/entities/benchmark_result.py

```diff
@@ -38,6 +38,7 @@ class BenchmarkResult(Base, EntityMixin):
     context = relationship("Context", lazy="joined")
     run = relationship("Run", lazy="select")
     unit = Nullable(s.Text)
+    language = Nullable(s.Text)
```

Comment:
Language is not benchmark result so it should not be stored in benchmark results